articles/libraries/auth0js/v8/index.md

```diff
@@ -483,6 +483,13 @@ If attempting to set up a password reset functionality, you'll use the `changePa
 
 The user will then receive an email which will contain a link that they can follow to reset their password.
 
+## Cross-Origin Authentication
+
+Using auth0.js within your application, rather than using the [Hosted Login Page](/hosted-pages/login), requires [cross-origin authentication](/cross-origin-authentication). In order to use embedded auth0.js via cross-origin authentication, you must do the following:
+
+* Set the [audience](/libraries/lock/v10/configuration#audience-string-) option
+* In the client settings area of the [Dashboard]($manage_url}), in the **Advanced Settings** menu, under the **OAuth** tab, turn on the OIDC Conformant and Cross Origin Authentication settings.
```

Comment:
These are fields so they should be formatted as bold: OIDC Conformant, Cross Origin Authentication

I'd also add a screenshots of these fields.